CMakePresets.json

```diff
@@ -1,21 +1,30 @@
 {
-    "version": 3,
-    "configurePresets": [
-        {
-            "name": "clang",
-            "generator": "Ninja",
-            "cacheVariables": {
-                "CMAKE_BUILD_TYPE": "Debug",
-                "CMAKE_EXPORT_COMPILE_COMMANDS": "ON"
-            }
-        },
-        {
-            "name": "gcc",
-            "generator": "Ninja",
-            "cacheVariables": {
-                "CMAKE_BUILD_TYPE": "Debug",
-                "CMAKE_EXPORT_COMPILE_COMMANDS": "ON"
-            }
-        }
-    ]
+ "version": 3,
+ "configurePresets": [
+  {
+   "name": "clang",
+   "generator": "Ninja",
+   "cacheVariables": {
+    "CMAKE_BUILD_TYPE": "Debug",
+    "CMAKE_EXPORT_COMPILE_COMMANDS": "ON"
+   }
+  },
+  {
+   "name": "gcc",
+   "generator": "Ninja",
+   "cacheVariables": {
+    "CMAKE_BUILD_TYPE": "Debug",
+    "CMAKE_EXPORT_COMPILE_COMMANDS": "ON"
+   }
+  },
+  {
+   "name": "debug",
+   "generator": "Ninja",
+   "cacheVariables": {
+    "CMAKE_BUILD_TYPE": "Debug",
+    "CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
+    "CMAKE_TOOLCHAIN_FILE": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"

```

Comment:
Don't just add this to the new config and not any of the old ones. These should be kept in sync.